scripts/benchmark_k.py

```diff
@@ -27,6 +27,7 @@
 def handleArgs(argv):
   data_name_options = ['zeisel', 'paul', 'cite_seq', 'mouse_brain']
   eval_model_options = ['random_forest', 'k_nearest_neighbors']
+  benchmark_options = ['k', 'label_error', 'label_error_markers_only']
```

Comment:
it doesn't seem that k refers to the definition used elsewhere? I am not sure if this is actually overloading the variable, but if it is, can we rename it? I see multiple ways to use k below, so I think I am misreading it.